src/modules/info/translations/en.json

```diff
@@ -36,9 +36,25 @@
             "select": {
                 "command": "Select a command"
             }
+        },
+        "avatar": {
+            "title": "Avatar of",
```

Comment:
El nombre de usuario va incluido en la frase, usa un parámetro como "avatar of {user}"